core/state/database.go

```diff
@@ -194,6 +197,9 @@ func (db *cachingDB) OpenTrie(root common.Hash) (Trie, error) {
 
 // OpenStorageTrie opens the storage trie of an account.
 func (db *cachingDB) OpenStorageTrie(stateRoot common.Hash, address common.Address, root common.Hash) (Trie, error) {
+	if db.triedb.IsZkTree() {
```

Comment:
Why don't we rename this to `IsZkStateTree()`?

Reply:
i'll change it to `IsZkStateTrie`.